lib/hexpm/web/controllers/package_controller.ex

```diff
@@ -2,7 +2,7 @@ defmodule Hexpm.Web.PackageController do
   use Hexpm.Web, :controller
 
   @packages_per_page 30
-  @sort_params ~w(name downloads inserted_at updated_at)
+  @sort_params ~w(name ninety_days total_downloads inserted_at updated_at)
```

Comment:
ninety_days is fine for internal use, but lets call it recent_downloads and total_downloads for users.

Reply:
Fixed.

Reply:
Updated to recent_downloads.